src/integrationTest/sam.test.ts

```diff
@@ -18,13 +18,14 @@ import { VSCODE_EXTENSION_ID } from '../shared/extensions'
 import { fileExists } from '../shared/filesystemUtilities'
 import { AddSamDebugConfigurationInput } from '../shared/sam/debugger/commands/addSamDebugConfiguration'
 import { findParentProjectFile } from '../shared/utilities/workspaceUtils'
-import { activateExtension, getCodeLenses, getTestWorkspaceFolder, sleep } from './integrationTestsUtilities'
+import * as testUtils from './integrationTestsUtilities'
```

Comment:
any reason this changed to *?

Reply:
Importing 7 different functions so it makes sense to import the whole module and give it a name.